ibis/pandas/aggcontext.py

```diff
@@ -359,67 +463,57 @@ def agg(self, grouped_data, function, *args, **kwargs):
                     raw=True,
                 )
         else:
-            # do mostly the same thing as if we did NOT have a grouping key,
-            # but don't call the callable just yet. See below where we call it.
+            # Get the DataFrame from which the operand originated
+            # (passed in when constructing this context object in
+            # execute_node(ops.WindowOp))
+            parent = self.parent
+            frame = getattr(parent, 'obj', parent)
+            obj = getattr(grouped_data, 'obj', grouped_data)
+            name = obj.name
```

Comment:
name could be None here (e.g. a grouped DataFrame), have you hit this case?

Reply:
grouped_data is either Series or SeriesGroupBy here

Reply:
Let me type that as well